packages/diagrams/frontend/sirius-components-diagrams/src/renderer/connector/useConnectorNodeStyle.tsx

```diff
@@ -43,5 +43,11 @@ export const useConnectorNodeStyle = (nodeId: string, descriptionId: string): Us
       style.opacity = '0.4';
     }
   }
-  return { style };
+
+  const memoizedStyle = useMemo(
+    () => style,
+    [candidates.map((candidate) => candidate.id).join('-'), isNewConnection, hoveredNode?.id]
+  );
+
+  return { style: memoizedStyle };
```

Comment:
typo `memoized` -> `memorized`

Reply:
Memoizing and memorizing are two different things, I think that Michael is correct here in mentioning memoizing. See the official documentation using this term: https://react.dev/reference/react/useMemo#memoizing-a-dependency-of-another-hook

Reply:
Or wikipedia: https://en.wikipedia.org/wiki/Memoization :)

Reply:
I didn't know the term, thanks !